aws_sra_examples/solutions/s3/s3_block_account_public_access/customizations_for_aws_control_tower/manifest_v2.yaml

```diff
@@ -34,7 +34,7 @@ resources:
       - parameter_key: pSRAAlarmEmail
         parameter_value: ''
       - parameter_key: pSRASolutionVersion
-        parameter_value: 'v1.2'
+        parameter_value: 'v1.3'
```

Comment:
Update line 75 in case someone uncomments and uses the ...main.yaml file

Reply:
Good catch!  I've supplied another update.